apps/src/main/resources/application.conf

```diff
@@ -0,0 +1,75 @@
+akka {
+  extensions = ["com.romix.akka.serialization.kryo.KryoSerializationExtension$"]
+
+  diagnostics.checker.confirmed-typos = [
+    "akka.actor.kryo"
+  ]
+
+  actor {
+    provider = "cluster"
+
+    allow-java-serialization = no
+
+    serializers {
+      kryo = "com.romix.akka.serialization.kryo.KryoSerializer"
+    }
+
+    serialization-bindings {
+      "com.lightbend.akka.bench.ddata.DDataHost$Added$" = kryo
+    }
+
+    # kryo for serialization for lazy reasons
+    kryo  {
+      type = "nograph"
+      idstrategy = "explicit"
+      buffer-size = 4096
+      max-buffer-size = -1
+      use-manifests = false
+      use-unsafe = false
+      post-serialization-transformations = "off"
+      implicit-registration-logging = false
+      kryo-trace = false
+
+      mappings {
+        "com.lightbend.akka.bench.ddata.DDataHost$Added$" = 20
+      }
+    }
+  }
+
+  remote {
+    artery {
+      enabled = on
+      canonical.hostname = "127.0.0.1"
+      canonical.port = 0
```

Comment:
this should bind to the IP and port provided by the root-application.conf - I'll do this